web/css/source/extend/_buttons.less

```diff
@@ -0,0 +1,8 @@
+& when (@media-common = true) {
+    button,
+    .action.primary,
+    .action-primary {
+        transition: @button__transition;
+    }
+
```

Comment:
try to use the same style before and after a block. here you have empty line, but before not